src/FormBuilder/resources/js/form-builder/src/blocks/fields/payment-gateways/settings.tsx

```diff
@@ -13,9 +13,13 @@ const settings: FieldBlock['settings'] = {
     },
     attributes: {
         lock: {remove: true},
+        gatewaysSettings: {
+            type: 'object',
+            default: {},
+        },
```

Comment:
hmm, I think this might be an early abstraction.  With a big gatewaySettings object, we get ourselves into a big unknown shape because all gateways are different.  We also risk a rogue gateway overriding the entire attribute object and causing issues for other gateways.  In that light, I would prefer specific gateway attributes with clear motives and types like this:

```
stripeUseGlobalDefaults: {
  type: 'boolean'
  default: true
}
```